src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/sendReport.hpp

```diff
@@ -83,14 +85,11 @@ class SendReport final : public AbstractHandler<std::shared_ptr<ScanContext>>
             }
         }
 
-        if (data->m_elements.empty())
-        {
-            return nullptr;
-        }
-
-        return AbstractHandler<std::shared_ptr<ScanContext>>::handleRequest(data);
+        return AbstractHandler<std::shared_ptr<TScanContext>>::handleRequest(data);
     }
     // LCOV_EXCL_STOP
```

Comment:
We are excluding `handleRequest()`.
The exclusion should only be for constructor/destructor. Otherwise, we must create a technical debt issue.

Reply:
Exclusion range was fixed